### PR TITLE
Release notes for Percona Server 5.7.17-12 (5.7)

### PR DESCRIPTION
**Pull request by hrvojem**

### CODE REVIEW

**Review by george-lorch — APPROVED** [review]
MyRocks and TokuDB LGTM

File: doc/source/flexibility/extended_mysqldump.rst [hunk 1]
```diff
@@ -36,6 +36,16 @@ implemented. This feature tells ``mysqldump`` to take the backup by
 descending primary key order (``PRIMARY KEY DESC``) which can be useful if
 storage engine is using reverse order column for a primary key.
 
+RocksDB support
+===============
+
+:command:`mysqldump` will now detect when MyRocks is installed and available
+by seeing if there is a session variable named
+:variable:`rocksdb_skip_fill_cache` and set it to ``1`` if exists.
```
Comment (laurynas-biveinis) [inline doc/source/flexibility/extended_mysqldump.rst]: s/set/setting
s/if exists/if it exists

File: doc/source/release-notes/Percona-Server-5.7.17-12.rst [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+ :variable:`rocksdb_skip_fill_cache` and set it to ``1`` if exists.
+
+ :command:`mysqldump` will now automatically enable session variable
+ :variable:`rocksdb_bulk_load` if it is supported by target server.
+
+Bugs Fixed
+==========
+
+ If the variable :variable:`thread_handling` was set to ``pool-of-threads`` in
+ the |MySQL| configuration file, server couldn't be gracefully shut down. Bug
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.7.17-12.rst]: shut down by a SIGTERM signal

File: doc/source/release-notes/Percona-Server-5.7.17-12.rst [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+ :bug:`1652852` (upstream :mysqlbug:`84350`).
+
+ If ``SHOW BINLOGS`` or ``PERFORMANCE_SCHEMA.GLOBAL_STATUS`` query, and a
+ transaction commit would run in parallel, they could deadlock. Bug fixed
+ :bug:`1657128`.
+
+ Log tracking initialization did not find last valid bitmap data correctly. Bug
+ fixed :bug:`1658055`.
+
+ Running the query that uses range scan with primary key could lead to a server
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.7.17-12.rst]: A query using range scan with a complex range condition could lead ...

File: doc/source/release-notes/Percona-Server-5.7.17-12.rst [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+ Row-based replication events were not reflected in ``Rows_updated`` fields in
+ the :ref:`user_stats` ``INFORMATION_SCHEMA`` tables. Bug fixed :bug:`995624`.
+
+ When ``DuplicateWeedout`` strategy was used for joins, use was not reported in
+ the query plan info output extension for the slow query log. Bug fixed
+ :bug:`1592694`.
+
+ A long-running binary log commit would block ``SHOW STATUS``, which in turn
+ could block a number of of other operations such as client connects and
+ disconnects. Bug fixed  :bug:`1646100`.
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.7.17-12.rst]: Move up next to 1657128

File: doc/source/release-notes/Percona-Server-5.7.17-12.rst [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+ It was impossible to use column compression dictionaries with partitioned
+ |InnoDB| tables. Bug fixed :bug:`1653104`.
+
+ ``FLUSH LOGS`` was disabled with :variable:`read_only` and
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.7.17-12.rst]: Move up next to 1652852

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**